.github/workflows/golf-integration-translations-received.yml

```diff
@@ -0,0 +1,32 @@
+name: GoLF Integration - Sync Translations Received from GoLF
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - translation
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current repository
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+        with:
+          fetch-depth: 0
+
+      - name: Checkout action-golf-integration repository
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
```

Comment:
let's use checkout v4 - the older version has been deprecated/support may be removed later (better to reference SHA than version number as in current code)